buildsystem/check_py_file_list.py

```diff
@@ -37,12 +37,12 @@ def main():
     success = True
     for filename in sorted(actual - listed):
         success = False
-        print("file was not listed via add_py_module: " +
+        print("file was not listed via add_py_module: " + filename + " as " +
               os.path.relpath(filename, cwd))
 
     for filename in sorted(listed - actual):
         success = False
-        print("file was listed via add_py_module but does not exist: " +
+        print("file was listed via add_py_module but does not exist: " + filename + " as " +
```

Comment:
I put them in for my debugging. I left them there because I thought they'd be useful to others too.